apps/ledger-live-mobile/src/components/RootNavigator/EarnLiveAppNavigator.tsx

```diff
@@ -21,9 +23,12 @@ const Earn = (_props: StackNavigatorProps<EarnLiveAppNavigatorParamList, ScreenN
   // TODO: update to use specific mobile feature flag
```

Comment:
if you go back to these files it'd be nice to remove the no longer relevant comment on line 23 btw (just an aside / nice to have)

Reply:
done 👍